File: backends/ruby-hanami/app/usecase/invitation/dto.rb
# frozen_string_literal: true
#
# 招待ユースケースの DTO を定義するモジュール。
#
# @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>

module UseCase
  module Invitation
    # トークンによる招待検索時に渡す DTO です。
    FindByTokenDto = Struct.new(:token, keyword_init: true)
  end
end

File: backends/kotlin-ktor/src/main/kotlin/com/authorization/usecase/invitation/Dto.kt
/*
 * 招待ユースケース DTO モジュール。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
package com.authorization.usecase.invitation

/**
 * 招待トークン検索 DTO です。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
data class FindByTokenDto(
    val token: String,
)
